asu/janitor.py

```diff
@@ -419,11 +419,18 @@ def update_meta_json():
         json.dumps(
             current_app.config["OVERVIEW"],
             indent=2,
+            sort_keys=False,
```

Comment:
Why not sort them? Isn't that better for reproducibility?

Reply:
I thought it might make more sense to have them in the same order like the branches.yml (if you have one) and also in the same order like the structure that is defined in the source code (`current_app.config["OVERVIEW"]`). But yes - we could sort them

Reply:
Good point. Arrays should be sorted at any point but dicts are.